apiary.apib

```diff
@@ -374,6 +374,8 @@ We list all backwards-compatible additions here. These are currently available i
 
 #### Latest
 
+- We exposed the "Freeze time tracking after a number of days" preference in `users.me`.
```

Comment:
Q: do we need to update the description to resemble the updated name?

Reply:
That's how it's called in the UI currently.

Reply:
![image](https://user-images.githubusercontent.com/10810561/74443434-d59cfa00-4e7b-11ea-974c-53f1e172b38f.png)

Reply:
Okay, clear enough than. 👍